components/list/test/list.visual-diff.js

```diff
@@ -15,7 +17,7 @@ describe('d2l-list', () => {
 	});
 
 	beforeEach(async() => {
-		await visualDiff.resetFocus(page);
+		await page.reload();
```

Comment:
Unsure how much this slows tests down - was the easiest way to restore things.

Reply:
I used reload because the `tooltip-helper` didn't have a nifty `reset` method. https://github.com/BrightspaceUI/core/blob/master/components/tooltip/test/tooltip.visual-diff.js#L19

I could write one, but it seems weird to write it within the list tests.

Reply:
Wow yeah, and quite a few tests as well.  That makes me wonder how much time is actually being spent on the page loads.  Tooltip does have public methods for hiding though... so seems possible to reset.

Reply:
It'd be nice to find a way to measure the page load time so we can say definitely what it costs us, and whether we should continue to avoid reloading or not.  Don't need to figure it our here/now, but would certainly be good to know going forward.

Reply:
I'll just write one in a separate PR.